### PR TITLE
Store and Use DIDs for Portal magics

### PR DESCRIPTION
**Pull request by LtRipley36706**
Instead of just saving positions, we need to store as DIDs and do proper lookups back to the original weenie for restrictions and other variables.

### CODE REVIEW

File: Source/ACE.Server/WorldObjects/Monster_Awareness.cs [hunk 1]
```diff
@@ -80,6 +80,7 @@ public virtual void HandleFindTarget()
         {
             return;
 
+            /*
```
Comment (mcreedjr) [inline Source/ACE.Server/WorldObjects/Monster_Awareness.cs]: If you don't mind my asking, would you mind sharing what error you resolved by commenting out this code block? I believe this is a fairly critical piece of the monster 'think' logic.

Comment (LtRipley36706) [reply Source/ACE.Server/WorldObjects/Monster_Awareness.cs]: there's a `return;` line just above the commenting so nothing is getting past that making this code useless. I wanted to clean up the existing warnings that have been lingering for months

Comment (mcreedjr) [reply Source/ACE.Server/WorldObjects/Monster_Awareness.cs]: Got it. I'm sorry. I didn't 'review' thoroughly enough to see that! Thank you for taking the time to clean up those lingering warnings at compile time!

Comment (LtRipley36706) [reply Source/ACE.Server/WorldObjects/Monster_Awareness.cs]: Not a problem!

Comment (gmriggs) [reply Source/ACE.Server/WorldObjects/Monster_Awareness.cs]: This is WIP monster aggro code... it's not being used yet, so it just returns immediately

Commenting it out for now is a good idea

**Pull request merged**